src/randoop/operation/MethodParser.java

```diff
@@ -0,0 +1,51 @@
+package randoop.operation;
+
+import java.lang.reflect.Method;
+
+import randoop.types.TypeNames;
+
+/**
+ * MethodParser provides methods to recognize and extract methods from
```

Comment:
If it's a library of static methods rather than representing anything, please state that explicitly here.